stl/inc/numbers

```diff
@@ -0,0 +1,188 @@
+// numbers standard header (core)
+
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#pragma once
+#ifndef _NUMBERS_
+#define _NUMBERS_
+#include <yvals_core.h>
+#if _STL_COMPILER_PREPROCESSOR
+#if !_HAS_CXX20
+#pragma message("The contents of <numbers> are available only with C++20 or later.")
+#else // ^^^ !_HAS_CXX20 / _HAS_CXX20 vvv
+#ifdef __cpp_lib_concepts
+#include <concepts>
+#endif
+
+#pragma pack(push, _CRT_PACKING)
+#pragma warning(push, _STL_WARNING_LEVEL)
+#pragma warning(disable : _STL_DISABLED_WARNINGS)
+_STL_DISABLE_CLANG_WARNINGS
+#pragma push_macro("new")
+#undef new
+
+_STD_BEGIN
+namespace numbers {
+    template <class _Ty>
+    struct _Invalid {
+        static_assert(_Always_false<_Ty>, "This variable template is not specialised for this type which is invalid.");
+    };

```

Comment:
I recommend adding a newline after this struct definition. It's purely stylistic, but we generally like to "take a breath" between different parts of code, and here you've defined the `_Invalid` helper before starting a repetitive series of definitions, which are related but different sections.